README.md

```diff
@@ -18,14 +18,15 @@ if you have not already.
 1. Set-up Google Cloud to deploy the scanner.
   1. Create a new project for the scanner by going to the
      Google Cloud Console and clicking on "Create Project".
-     Note the project ID of this project.
+     Note the project number and name of this project.
```

Comment:
nit: I would keep the "project id" rather than "project number".  Because "project id" is more consistent with the documentation, and there's no pure number used.